### PR TITLE
Adding vShpere volume section to Volumes doc

### PR DESCRIPTION
**Pull request by dagnello**
This documentation is being added in reference to vSphere cloud provider
support work (issue [#23932](https://github.com/kubernetes/kubernetes/issues/23932)).

### CODE REVIEW

File: docs/user-guide/volumes.md [hunk 1]
```diff
@@ -379,6 +380,46 @@ into a Pod.
...
 More details can be found [here](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/azure_file/README.md)
 
+### vsphereVirtualDisk
+
+A `VSphereVirtualDisk` is used to mount a vSphere VMDK Volume into your Pod.  The contents
+of a volume are preserved when it is unmounted.
+
+__Important: You must create a VMDK volume using `vmware-vdiskmanager -c` or
+the VSphere API before you can use it__
+
```
Comment (johndmulhausen) [inline docs/user-guide/volumes.md]: Can we remove two of these blank lines?

Comment (dagnello) [inline docs/user-guide/volumes.md]: updated

**Comment by dagnello** [comment]
These Doc updates are associated with the following PR:
[vSphere Volume Plugin Implementation](https://github.com/kubernetes/kubernetes/pull/24947)

**Comment by dagnello** [comment]
@johndmulhausen the respective PR has now merged (PR [24947])(https://github.com/kubernetes/kubernetes/pull/24947).  Can we get this merged as well to keep docs updated?

**Comment by johndmulhausen** [comment]
@lavalamp @thockin - you helped @dagnello with the code PR. Do the docs here look OK?

**Comment by lavalamp** [comment]
lgtm

**Comment by dagnello** [comment]
@johndmulhausen good to merge

**Comment by dagnello** [comment]
@johndmulhausen is there anything else remaining for this PR?

**Comment by dagnello** [comment]
@johndmulhausen is there anything else remaining for this PR?

**Comment by dagnello** [comment]
@johndmulhausen can we get this merged?

**Pull request merged**